pkg/apiserver/api_installer.go

```diff
@@ -139,7 +139,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	// we're trying to register here
 	fqKindToRegister := unversioned.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
-		if fqKind.Group == a.group.GroupVersion.Group {
+		if fqKind.Group == a.group.GroupVersion.Group || (fqKind.Group == extensions.GroupName && fqKind.Kind == "ScaleTwo") {
```

Comment:
@lavalamp @deads2k @caesarxuchao trying to remember which we were leaning toward